libs/slam/slam/loop_closure_solver/lcs_simple.cpp

```
@@ -210,6 +211,11 @@ void LoopClosureSolverSimple::SelectLandmarksCandidates(const LoopClosureTask& t
     for (LandmarkId id : lsi_candidate_ids) {
       const Vector3T xyz = landmarks_spatial_index.GetLandmarkOrStagedCoords(id, *task.pose_graph_hypothesis);
       const Vector3T point_guess = camera_from_world_guess * xyz;
+      // same near plane the triangulator and the visibility test use — a landmark nearer than this
```

Cppcheck (2.21.0)

[style] The function 'IsApprox' is never used.

(unusedFunction)